src/Site/Views/Releases/Show.cshtml

```diff
@@ -1,6 +1,6 @@
 @model RimDev.Releases.ViewModels.Releases.ShowViewModel
 @{
-    ViewData["Title"] = "Home Page";
+    ViewData["Title"] = $"Releases for `{Model.CurrentRepository.Description}`";
```

Comment:
The intent is to separate the description of the repo. from the body of the title. Up for suggestions for another character.